openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -439,10 +439,35 @@ class c02900(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
+    label = "'Above the line' AGI deductions"
+    unit = USD
     documentation = (
         """Total of all 'above the line' income adjustments to get AGI"""
     )
 
+    def formula(tax_unit, period, parameters):
+        misc_haircuts = parameters(period).tax.ald.misc
+        return sum(
+            [
+                (1 - haircut) * tax_unit(variable, period)
+                for haircut, variable in (
+                    (misc_haircuts.c03260, "c03260"),
```

Comment:
Do `c03260` and `care_deduction` not apply the filer logic (i.e. not `filer_c03260`)?

Reply:
They don't as they're already tax-unit level